src/app/settings.ts

```diff
@@ -157,6 +157,28 @@ export class Settings {
   rGlobe: 0;
   lastTime: 0;
 
+  levelCheck(str) {
+    if (str === "Default") {
```

Comment:
I'm not a fan of this hack either. We should just change this in the database instead.

Reply:
If you want to keep it here, add a `TODO:` comment so we know to eventually fix this in the database.